### PR TITLE
Adds a mining shuttle console and ore box to aux mining bases on all …

### PR DESCRIPTION
**Pull request by WJohn**
...stations.

Exactly what it says on the tin. Pleeeeaaase don't make me have to give you pics for this it's so darn minor.

Touches box, meta, delta, pubby, and cere -- omega doesn't have an aux base.

:cl: optional name here
add: Added mining shuttle consoles and ore boxes to aux mining bases on all stations.
/:cl:

### CODE REVIEW

**Comment by GunHog** [comment]
Yaaay :D

**Comment by PraiseRatvar** [comment]
best changelog

File: _maps/map_files/PubbyStation/PubbyStation.dmm [hunk 1]
```diff
@@ -47816,6 +47816,7 @@
 /obj/effect/turf_decal/stripes/line{
 	dir = 9
 	},
+/obj/structure/closet/secure_closet/miner,
```
Comment (ShizCalev) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: /obj/structure/closet/secure_closet/miner/unlocked

**Comment by Cyberboss** [comment]
Reee changelog!!!

File: _maps/map_files/PubbyStation/PubbyStation.dmm (hunk 1 above)
Comment (Supermichael777) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: negative

**Comment by Repukan** [comment]
Pics pls

File: _maps/map_files/PubbyStation/PubbyStation.dmm (hunk 1 above)
Comment (ShizCalev) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: @Supermichael777 Why negative? That's the path of the unlocked locker that's present in every aux_base area.

Comment (GunHog) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: That is correct. The mining locker in ALL Aux Bases should be unlocked. It is intended to let other crew experience Lavaland, OR be a second base the miners can use if a Drake decided to eat the primary base.

**Comment by The-clown-known-as-Pepe** [comment]
@WJohn the computer in the middle acts like shuttle console when beacon is set up, there is no need for another console

**Comment by GunHog** [comment]
>@WJohn the computer in the middle acts like shuttle console when beacon is set up, there is no need for another console

It needs to be OUTSIDE of the Aux Base, in the _construction_ area. The idea is so that people at the public dock can bring the mining shuttle to where the base used to be.

**Comment by WJohn** [comment]
I'll just redo it, sorry about that

**Pull request closed without merging**